server.js

```diff
@@ -1,16 +0,0 @@
-const express = require("express");
-const app = express();
-const port = 3000;
-const studentRoutes = require("./src/students/routes");
-
-app.get("/" , (req,res) => {
-    res.send("Hello World");
-});
-
-app.use(express.json());
-
-app.use("/students", studentRoutes)
```

Comment:
How to serve this API only when authenticated and otherwise redirect to signup page